src/Token.php

```diff
@@ -51,5 +51,10 @@ public function isExpired(DateTimeInterface $now): bool;
     /**
      * Returns an encoded representation of the token
      */
+    public function toString(): string;
+
+    /**
+     * Returns an encoded representation of the string while casting Token to string
+     */
     public function __toString(): string;
```

Comment:
Let's really drop it.
On v3.4 we mark it as `@deprecated` and introduce `toString()` as well.

Reply:
was worried it wouldn't be approved if I drop it :) cheers, will update now.

Reply:
Thanks, it was a bad decision back then. I just had bigger fishes to fry :sweat_smile: